Tests/PointFreeTests/__Snapshots__/SiteMiddlewareTests/testWithHttps.localhost_allowed.Conn.txt

```diff
@@ -0,0 +1,2821 @@
+                </label>
+                <span class="m-pl2
+                             m-pr2
+                             fg-gray850">
+                  or
+                </span>
+                <input checked
+                       class="none"
+                       id="tab1"
+                       name="pricing[lane]"
+                       type="radio"
+                       value="team"
+                       role="button">
+                <label for="tab1"
+                       class="fit
+                              fg-black
+                              bold
+                              ts-d-r1_25
+                              ts-m-r1
+                              lh-1
+                              medium
+                              m-pl2
+                              m-pr2
+                              m-pt2
+                              m-pb2
+                              border-none
+                              inline-block
+                              rounded
+                              cursor-pointer
+                              bg-purple
+                              fg-white"
+                       style="-webkit-box-shadow:0px 0px 5px 5px rgba(0,0,0,0.1);
+                              -moz-box-shadow:0px 0px 5px 5px rgba(0,0,0,0.1);
+                              -ms-box-shadow:0px 0px 5px 5px rgba(0,0,0,0.1);
+                              -o-box-shadow:0px 0px 5px 5px rgba(0,0,0,0.1);
+                              box-shadow:0px 0px 5px 5px rgba(0,0,0,0.1);
+                              width:35%">
+                  For your team
+                </label>
+                <div class="m-mb3">
+                </div>
+                <div class="flex">
+                  <div class="content1
+                              bg-white
+                              w-100p
+                              row">
+                    <div class="col-m
+                                col-m-12">
+                      <div class="m-pl3
+                                  m-pr3
+                                  m-pt3">
+                        <p class="fg-black
+                                  fg-black
+                                  normal
+                                  h5
+                                  lh-4">
+                          How many in your team?
+                        </p>
+                        <input class="num-spinner
+                                      center
+                                      fg-black
+                                      bold
+                                      ts-d-r4
+                                      ts-m-r3
+                                      lh-2
+                                      fg-black"
+                               max="100"
+                               min="2"
+                               name="pricing[quantity]"
+                               onblur="javascript:
+this.value
```

Comment:
this is funny, we should double check what the pretty printer is doing with this attribute...